src/__test__/test-helper.jsx

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+import renderer from 'react-test-renderer';
+
+const shouldMatchSnapshot = (ComponentName, Component) => {
+  describe(`"${ComponentName}"`, () => {
```

Comment:
We should add this wrapping describe block in the calling tests, not this function. Please remove it, and the corresponding `ComponentName` parameter.